Null reference while trying to run

I get the following exception when doing `mvn compile package`

<pre>Exception in thread &quot;main&quot; java.lang.AssertionError: A reference was unexpectedly null.
	at org.moditect.internal.shaded.javaparser.utils.Utils.assertNotNull(Utils.java:47)
	at org.moditect.internal.shaded.javaparser.Providers.provider(Providers.java:78)
	at org.moditect.internal.shaded.javaparser.JavaParser.parse(JavaParser.java:305)
	at org.moditect.internal.shaded.javaparser.StaticJavaParser.parse(StaticJavaParser.java:229)
	at org.moditect.internal.compiler.ModuleInfoCompiler.parseModuleInfo(ModuleInfoCompiler.java:68)
	at org.moditect.mavenplugin.add.AddModuleInfoMojo.getAssignedModuleNamesByModule(AddModuleInfoMojo.java:390)
	at org.moditect.mavenplugin.add.AddModuleInfoMojo.execute(AddModuleInfoMojo.java:133)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:137)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:210)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:156)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:148)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:117)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:81)
	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:56)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:305)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:192)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:105)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:956)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:288)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:192)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
</pre>

with the following POM.xml:
```xml
	        <plugin>
	            <groupId>org.moditect</groupId>
				<artifactId>moditect-maven-plugin</artifactId>
				<version>1.0.0.RC1</version>
				<executions>
					<execution>
						<id>generate-module-info</id>
						<phase>package</phase>
						<goals>
							<goal>add-module-info</goal>
						</goals>
						<configuration>
						    <outputDirectory>${project.build.directory}/modules</outputDirectory>
							<modules>
								<module>
									<artifact>
										<groupId>org.sql2o</groupId>
										<artifactId>sql2o</artifactId>
										<version>1.6.0</version>
									</artifact>
									<artifact>
										<groupId>tech.tablesaw</groupId>
										<artifactId>tablesaw-core</artifactId>
										<version>0.38.1</version>
									</artifact>
								</module>
							</modules>
						</configuration>
					</execution>
				</executions>
	        </plugin>
```

Can't figure out if I'm missing something or it's a bug

--- a/core/src/main/java/org/moditect/commands/CreateRuntimeImage.java
+++ b/core/src/main/java/org/moditect/commands/CreateRuntimeImage.java
@@ -50,7 +50,7 @@ public class CreateRuntimeImage {
     private boolean ignoreSigningInformation;
     private final String launcher;
     private final Log log;
-    private final Integer compression;
+    private final String compression;
     private final boolean stripDebug;
     private final boolean noHeaderFiles;
     private final boolean noManPages;
@@ -59,7 +59,7 @@ public class CreateRuntimeImage {
 
     public CreateRuntimeImage(Set<Path> modulePath, List<String> modules, JarInclusionPolicy jarInclusionPolicy,
                               Set<Path> dependencies, Path projectJar, String launcherName, String launcherModule,
-                              Path outputDirectory, Integer compression, boolean stripDebug,
+                              Path outputDirectory, String compression, boolean stripDebug,
                               boolean ignoreSigningInformation, List<String> excludeResourcesPatterns, Log log,
                               boolean noHeaderFiles, boolean noManPages, boolean bindServices) {
         this.modulePath = (modulePath != null ? modulePath : Collections.emptySet());
@@ -175,7 +175,7 @@ public class CreateRuntimeImage {
 
         if (compression != null) {
             command.add("--compress");
-            command.add(compression.toString());
+            command.add(compression);
         }
 
         if (stripDebug) {

--- a/maven-plugin/src/main/java/org/moditect/mavenplugin/image/CreateRuntimeImageMojo.java
+++ b/maven-plugin/src/main/java/org/moditect/mavenplugin/image/CreateRuntimeImageMojo.java
@@ -76,7 +76,7 @@ public class CreateRuntimeImageMojo extends AbstractMojo {
     private Launcher launcher;
 
     @Parameter
-    private Integer compression;
+    private String compression;
 
     @Parameter(defaultValue = "false")
     private boolean stripDebug;

